src/client/routes/manage/HackerView.tsx

```diff
@@ -103,11 +103,11 @@ export const HackerView: React.FunctionComponent<RouteComponentProps<{}>> = (
 				}
 
 				if (error) {
-					console.log(error);
+					// console.log(error);
 					return <GraphQLErrorMessage text={STRINGS.GRAPHQL_ORGANIZER_ERROR_MESSAGE} />;
 				}
 
-				console.log(data);
+				// console.log(data);
```

Comment:
ok I could have sworn the original version of this pr used pino everywhere

Reply:
can't use pino client side